mapping/models.py

```diff
@@ -1,3 +1,38 @@
 from django.db import models
 
 # Create your models here.
+
+class Country(models.Model):
+    name = models.CharField(max_length=50)
+    # add other fields/metadata here 
+    
+    def __str__(self):
+        return self.name
+
+class TaskDir(models.Model):
+    DEPARTMENT = (
+        ('Health'),
+        ('Organisation Development', 'OD'),
+        ('Disaster Management', 'DM'),
```

Comment:
Reverse the sequence i.e ('DM', 'Disaster Management') and provide the second field for 'Health'